CONTRIBUTING.md

```diff
@@ -304,6 +314,55 @@ If a site is available globally apart from a specific region, this can be noted
+| finance          |          Finance          | Financial, insurance and pension services                                        |
+| food             |           Food            | Food and beverage services                                                       |
+| gaming           |          Gaming           | Games and game platforms. Sites for buying games should be listed in Retail      |
+| government       |        Government         | Government portals. Excluding education                                          |
+| health           |          Health           | Health and fittness platforms                                                    |
+| hosting          |        Hosting/VPS        | Online website hosting, VPS and dedicated server rentals                         |
+| hotels           | Hotels and Accommodations | Hotels and short term accommodation providers                                    |
+| identity         |    Identity Management    | Authentication providers, Single Sign On platforms                               |
+| investing        |         Investing         | Investment platforms                                                             |
+| iot              |            IoT            | Internet of Things and device management platforms                               |
+| legal            |           Legal           | Legal aid services                                                               |
+| marketing        |   Marketing & Analytics   | Marketing campaign providers and analytics services                              |
+| payments         |         Payments          | Payment providers                                                                |
+| post             |     Post and Shipping     | Postal and logistics providers                                                   |
+| remote           |       Remote Access       | Remote device access platforms                                                   |
+| retail           |          Retail           | Online retail platforms                                                          |
+| security         |         Security          | Online security and anti-malware services. Excluding VPN and identity management |
+| social           |          Social           | Social networks                                                                  |
+| task             |      Task Management      | Task Management and ToDo-planning services                                       |
+| transport        |         Transport         | Transportation services including public transport and airlines                  |
+| universities     |       Universities        | University online platforms                                                      |
+| utilities        |         Utilities         | Household utilities including electricity,gas,water,phone and internet providers |
```

Comment:
Is the lack of a space after the commas to keep the column width fixed? Because it looks off to me.